content/releases/q/release-3.15.0.md

```diff
@@ -0,0 +1,14 @@
+---
+url: "/releases/q-3.15.0/"
+date: 2024-09-23
+eol: 2025-09-22
```

Comment:
Could it be min(camel_lts,quarkus_lts) = min(10-09-2025,2x-09-2025) = 10-09-2025 ?